PointerStar/Client/ViewModels/ClipboardService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.JSInterop;
+
+namespace PointerStar.Client.ViewModels;
+public interface IClipboardService
+{
+    ValueTask CopyToClipboard(string text);
+}
+
+public class ClipboardService : IClipboardService
+{
+    private readonly IJSRuntime _jsInterop;
+    public ClipboardService(IJSRuntime jsInterop) => _jsInterop = jsInterop;
+    public async ValueTask CopyToClipboard(string text) => await _jsInterop.InvokeVoidAsync("navigator.clipboard.writeText", text);
```

Comment:
This is one spot where you can get a minor optimization by omitting `async` and `await`
```suggestion
    public ValueTask CopyToClipboard(string text) =>  _jsInterop.InvokeVoidAsync("navigator.clipboard.writeText", text);
```